ios/jailed/Tweak.xm

```diff
@@ -0,0 +1,40 @@
+#import "lib/Cycript.framework/Headers/Cycript.h"
+
+%ctor {
+	CYListenServer(31337);
```

Comment:
Any reason why we’re launching Cycript here? Useful for sure, but I don’t think it should be part of the template. _Could_ instead be inserted by Theos when DEBUG=1 (before we reset it to 0).